site/content/blog/2023-07-05-zap-updates-june-2023/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "ZAP Updates - June 2023"
+summary: >
+  June 2023 updates and ongoing feature development statuses.
+images:
+- https://www.zaproxy.org/blog/2023-07-05-zap-updates-june-2023/images/zapbot-monthly-updates-social.png
+type: post
+tags:
+- blog
+- update
+date: "2023-07-05"
+authors: 
+- akshath
+---
+
+Welcome to our monthly updates blog post!  
+There were a lot of incremental updates to ZAP in June 2023.
+
+## Highlights
+### Browser-Based Authentication Support in the AJAX Spider
+The AJAX Spider now supports crawling authenticated web pages with the browser-based authentication method.
+ZAP now runs regularly scheduled scans against OWASP Juice Shop that uses this mode of authentication. You can view more details about the scan [here](/docs/scans/juiceshop/).
+
+### Selenium 4
+All ZAP add-ons are now using [Selenium](https://www.selenium.dev/) v4.
+
+### Docker images on GHCR
+We now publish all our Docker images on GitHub Container Registry, along with DockerHub.
+While anyone can use these images, we expect these to be useful for large organizations which may be hitting DockerHub's image pull limits.
+Read the [blog post](/blog/2023-06-13-ghcr-docker-images) for more details.
+
+### Community Scripts Tips (and Tricks)
+A new section was added to the community-scripts repository for people to share their ideas and usage tips for ZAP and its add-ons.
+Visit the [community-scripts repository](https://github.com/zaproxy/community-scripts/tree/main/other/tips) to learn more.
+
+## Ongoing Work
+### 2.13.0 Release
+We are actively working on getting ZAP 2.13.0 ready for release.
```

Comment:
"You can track the progress on https://github.com/zaproxy/zaproxy/issues/7924" ?